code/lib/builder-vite/src/utils/without-vite-plugins.ts

```diff
@@ -0,0 +1,16 @@
+import { PluginOption } from 'vite';
+
+// recursively remove all plugins with the given names
+export const withoutVitePlugins = (
+  plugins: PluginOption[] = [],
+  namesToRemove: string[]
```

Comment:
I think you've unintentionally helped me add another feature which I've had in mind, which is allowing users to remove plugins from their vite config by providing a list of names, similar to what Histoire allows: https://histoire.dev/guide/config.html#ignoring-plugins.  Having this function will come in handy.

Reply:
Ha great. I just thought that I'd written it 3 times now that it probably deserves it's own utility.